README.md

```diff
@@ -49,3 +49,28 @@ Legend:
   the subprocess provides data back to the process via pipes.
 - A dotted line indicates data movement between two processes.
 ![soft-t-ue.png](https://github.com/oran-testing/soft-t-ue/blob/main/docs/images/soft-t-ue.png)
+
+## Installation
+
+To install the UE run:
+`./scripts/install-ue.sh`
+
+To install the gNB run:
+```
+./scripts/install-open5gs.sh
+./scripts/install-gnb.sh
+```
+
+To install dependencies for the UE controller:
+```
+cd ue-controller
+poetry install
+```
+
+## Running 
+
+To run the controller:
+```
+cd ue-controller
```

Comment:
Can we provide a script for running this as well? It would be nice since we may want to export some ENV variable through to the process. What do you think?

Reply:
That's a good idea. But lets implement that once the system gets more complex. We will also need to start the gnb controller, and I think I may make another python file that does this and runs the app. For now just using poetry is enough.

Reply:
> That's a good idea. But lets implement that once the system gets more complex. We will also need to start the gnb controller, and I think I may make another python file that does this and runs the app. For now just using poetry is enough.

Fair enough but the gnb controller should be separate and running on another machine entirely so both controllers will not be able to be scripted together.